kani-compiler/src/codegen_cprover_gotoc/compiler_interface.rs

```diff
@@ -339,6 +339,17 @@ fn print_report(ctx: &GotocCtx, tcx: TyCtxt) {
         details.";
         tcx.sess.warn(&msg);
     }
+
+    if !ctx.concurrent_constructs.is_empty() {
+        let mut msg = String::from(
+            "Kani does not support concurrency for now. The following constructs will be treated \
```

Comment:
I know this message was there from before, but I suggest we change it to
```suggestion
            "Kani currently does not support concurrency. The following constructs will be treated \
```